doc/nrf/app_dev/device_guides/coprocessors/ug_hpf_softperipherals_comparison.rst

```diff
@@ -0,0 +1,124 @@
+     - Allows for the creation of specialized or custom peripherals.
+   * - Use cases
+     - When an application requires an additional or unavailable peripheral and a corresponding Soft Peripheral is available.
+     - When developing a custom peripheral or when high-degree control is needed.
+   * - Suitability
+     - Suitable for applications built from pre-compiled sources.
+     - Ideal for applications where the FLPR handles higher layers of the software stack.
+
+.. _nrf54l_hpf_softperi_comparison_requirements:
+
+Requirements
+************
+
+For Soft Peripherals, there is a specific memory size requirement of approximately 16 K, but there is a flexibility in placement within the FLPR execution RAM.
+To enter low power consumption modes, Soft Peripherals require a slot at a specific memory address, which is platform-specific, and the application code must be able to access MEMCONF registers.
+See the requirements in the :ref:`Soft Peripherals section<nrfxlib:soft_peripherals>`.
+It provides essential information for proper integration and optimization.
+
+On the other hand, HPF necessitates that the application core uses the Zephyr operating system.
+The framework allows for a high degree of configurability, which you can tailor according to your needs using Kconfig and devicetree settings.
+This allows you to optimize the memory footprint and functionality of the custom peripherals, ensuring that the peripherals are well-integrated and perform efficiently within their specific application environments.
+
+.. _nrf54l_hpf_softperi_comparison_features:
+
+Features comparison
+*******************
+
+See the following detailed feature comparison between Soft Peripherals and High-Performance Framework:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Feature
+     - Soft Peripherals
+     - High Performance Framework
+   * - Integration
+     - Pre-compiled binary for FLPR, driver for the application core
+     - Built from source in Zephyr build system
+   * - Memory requirements
+     - Fixed (~16 K)
+     - Adjustable, depends on code and devicetree
+   * - Application compatibility
+     - Can be used on baremetal or Zephyr applications
+     - Uses Zephyr without kernel on FLPR side, full Zephyr on APP side
+   * - Integration complexity
+     - Minor integration required
+     - Samples provided for the start of development
+   * - IPC mechanism
+     - Register-based
+     - ic(b)msg or mbox-based
+   * - Compliance and testing
+     - Protocol-compliant, verified by Nordic Semiconductor
+     - Samples provided as-is, testing is done by the user
+   * - Delivery form
+     - Binary
+     - Source code, modifiable and extendable
+   * - API level
+     - Exposed at hardware driver level
+     - FLPR may handle parts of the software stack above hardware driver level
+
+.. _nrf54l_hpf_softperi_comparison_creating_peripherals:
+
+Creation of software-based peripherals
```

Comment:
I'm a bit confused given that "soft peripherals", "software-defined peripherals", and "software-based peripherals" are all used on one page. Are they all the same? ;)